tests/product/base_product_case.py

```diff
@@ -124,16 +122,25 @@ def setup_docker_cluster(self, cluster_type='centos'):
 
     def tearDown(self):
         self.restore_stdout_stderr_keep_open()
```

Comment:
Since the dist directory is associated with a given DockerCluster object, that directory gets cleaned up in _tear_down_container().